process_report/invoices/NERC_total_invoice.py

```diff
@@ -60,4 +60,5 @@ def _prepare_export(self):
         ]
         self.data = self.data[
             self.data[invoice.INSTITUTION_FIELD].isin(self.INCLUDED_INSTITUTIONS)
+            | (self.data[invoice.GROUP_MANAGED_FIELD] == True)  # noqa E712
```

Comment:
Is this loaded as a boolean from the csv?

Reply:
If the CSV file has them as `"yes"`, `"no"` and this is the only place they're being used, then a comparison with `.lower() == "yes"` is enough.